cumulusci/tasks/datadictionary.py

```diff
@@ -4,7 +4,7 @@
 
 from distutils.version import LooseVersion
 
-
+from cumulusci.core.utils import process_bool_arg
```

Comment:
Please remove this import which you're no longer using.

And please make sure your editor runs flake8 on save, which will warn you about things like this.